tests/test_violations/test_codes.py

```diff
@@ -0,0 +1,42 @@
+from collections import Counter
```

Comment:
Should [these](https://github.com/wemake-services/wemake-python-styleguide/blob/master/tests/test_violations/test_codes.py#L26-L45) tests be adopted as well? `all_module_violations` mapping is not sorted and violation codes are not sequential

See `all_violation_codes` output
```
{
      <module 'dotenv_linter.violations.parsing' from '/home/outreach/Projects/dotenv-linter/dotenv_linter/violations/parsing.py'>: {
          1: <class 'dotenv_linter.violations.parsing.ParsingViolation'>,
      },
      <module 'dotenv_linter.violations.assigns' from '/home/outreach/Projects/dotenv-linter/dotenv_linter/violations/assigns.py'>: {
          200: <class 'dotenv_linter.violations.assigns.SpacedAssignViolation'>,
      },
      <module 'dotenv_linter.violations.comments' from '/home/outreach/Projects/dotenv-linter/dotenv_linter/violations/comments.py'>: {
          400: <class 'dotenv_linter.violations.comments.SpacedCommentViolation'>,
      },
      <module 'dotenv_linter.violations.names' from '/home/outreach/Projects/dotenv-linter/dotenv_linter/violations/names.py'>: {
          100: <class 'dotenv_linter.violations.names.SpacedNameViolation'>,
          101: <class 'dotenv_linter.violations.names.IncorrectNameViolation'>,
          102: <class 'dotenv_linter.violations.names.DuplicateNameViolation'>,
          103: <class 'dotenv_linter.violations.names.RawNameViolation'>,
          104: <class 'dotenv_linter.violations.names.ReservedNameViolation'>,
      },
      <module 'dotenv_linter.violations.values' from '/home/outreach/Projects/dotenv-linter/dotenv_linter/violations/values.py'>: {
          300: <class 'dotenv_linter.violations.values.SpacedValueViolation'>,
          301: <class 'dotenv_linter.violations.values.QuotedValueViolation'>,
      },
  }
```